docs/user-guide/use-certificates.md

```diff
@@ -0,0 +1,44 @@
+# Use certificates
+
+Now you can use the certificates for your next actions. The following certificates are supported by Zowe:
```

Comment:
Describe the precondition to use certificates. For example, "Once your keystore and truststore are configured and you have a valid certificate with teh certificate's corresponding private key..."